lib/services/messenger/translators/flowdock.ts

```diff
@@ -383,27 +384,34 @@ export class FlowdockTranslator extends TranslatorScaffold implements Translator
 	 * @param message  object to analyse.
 	 * @returns        Promise that resolves to emit instructions.
 	 */
-	private static searchThreadIntoEmit(message: TransmitInformation): Promise<FlowdockEmitInstructions> {
+	private static readThreadIntoEmit(message: TransmitInformation, term?: string): Promise<FlowdockEmitInstructions> {
 		// Check we have a thread.
 		const threadId = message.target.thread;
 		if (!threadId) {
 			return Promise.reject(new TranslatorError(
-				TranslatorErrorCode.IncompleteTransmitInformation, 'Cannot search for connections without a thread.'
+				TranslatorErrorCode.IncompleteTransmitInformation, 'Cannot read a thread without a thread.'
 			));
 		}
+		const payload: Dictionary<string> = {
```

Comment:
Just a comment, but given the object property `payload` being returned isn't actually the same `payload` being defined, maybe `body`?

Else I'd personally be expecting:
```typescript
return Promise.resolve({ method: ['get'], payload });
```